app/assets/javascripts/short-form/ShortFormApplicationController.js.coffee

```diff
@@ -965,17 +980,20 @@ ShortFormApplicationController = (
       if $scope.appIsSubmitted(previousApp)
         # My Applications page is now in React, prevent the "Leave Site?" popup when redirecting
         $window.removeEventListener('beforeunload', ShortFormApplicationService.onExit)
+        $window.removeEventListener('unload', $scope.onUnload)
         doubleSubmit = !! $scope.appIsSubmitted($scope.application)
         if $window.ACCOUNT_INFORMATION_PAGES_REACT is "true"
           currentUrl = window.location.origin
           newUrl = "#{currentUrl}/my-applications?"
           if previousApp.id
+            AnalyticsService.trackApplicationAbandon($scope.listing.Id, AccountService.loggedInUser?.id || null, 'Already Submitted')
```

Comment:
Same here, why would the id ever be null?

Reply:
It shouldn't ever be null, I added the check just to be 100% sure